.rubocop.yml

```diff
@@ -0,0 +1,84 @@
+inherit_from: .rubocop_todo.yml
+
+plugins:
+  - rubocop-rake
+  - rubocop-rspec
+
+AllCops:
+  NewCops: enable
+
+# Gemspec
+
+Gemspec/OrderedDependencies:
+  # Sort the gemspec dependencies by logical order.
+  Enabled: false
+
+# Layout
+
+Layout/FirstHashElementIndentation:
+  EnforcedStyle: consistent
+
+Layout/SpaceInsideArrayPercentLiteral:
+  # Sometimes spacing is used to align amounts.
+  Enabled: false
+
+Layout/SpaceInsideHashLiteralBraces:
+  # Prefer `{a: 1}` to `{ a: 1 }`
+  EnforcedStyle: no_space
```

Comment:
could we enforce the space?! 😳 I do have a strong preference for that and it's in the default ruby style guide https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#spaces-and-braces 😇